src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/config/executors/CcdEventExecutor.java

```diff
@@ -25,7 +25,7 @@ public class CcdEventExecutor {
     private int concurrentSessions;
 
     @Autowired
-    private CcdEventConsumer serviceBusTask;
+    private CcdCaseEventsConsumer serviceBusTask;
```

Comment:
This is wrong. You already have another executor to start CcdCaseEventConsumer. As we don't need CcdEventConsumer, you need to remove this class. Not changing it to start the CcdCaseEventConsumer as there will be two executors to start the same consumer.